examples/shared_samples/hypervisor_manager.rb

```diff
@@ -22,6 +22,10 @@
 # api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::HypervisorManager
 # api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::HypervisorManager
 
+@hypervisor_manager_ip= "172.18.13.11"
```

Comment:
this is already configured in _client.rb.example file

Reply:
@soodpr Changes Updated